lib/inets/test/httpc_SUITE.erl

```diff
@@ -381,10 +382,19 @@ get_space() ->
     [{"Test http get request with '%20' in the path of the URL."}].
 get_space(Config) when is_list(Config) ->
     Request  = {url(group_name(Config), "/space%20.html", Config), []},
-    {ok, {{_,200,_}, [_ | _], Body = [_ | _]}} = httpc:request(get, Request, [], []),
+    {ok, {{_,201,_}, [_ | _], Body = [_ | _]}} = httpc:request(get, Request, [], []),
```

Comment:
that was changed to 201, default response has same values
this can lead to hidden errors